doc/QuickStart.md

```diff
@@ -29,23 +32,48 @@ Installing
 
 ### Prerequisites
 
-#### Installing CPAN modules
+#### Requirements *(root)*
+
+	$ dnf -y install perl perl-CPAN git screen crontabs nano
+> or 		$ aptitude install perl perl-CPAN git screen crontabs nano
+> or 		$ apt-get install perl perl-CPAN git screen crontabs nano
+
+	$ adduser -m pbot
+	
+
+#### Installing CPAN modules *(root)*
 
 PBot has many features; some of these depend on Perl modules written by others.
-This list can be found in the `MODULES` file in the root directory of this source.
+This list can be found in the [`MODULES`](https://github.com/pragma-/pbot/blob/master/MODULES) file in the root directory of this source.
 
 The modules may be installed with a simple command:
 
-    cpan -f -i $(cat MODULES)
+    $ cpan
+	cpan[1]> force install <modules> 
+	cpan[49]> exit
```

Comment:
Why did you change this from the simpler one-line command that pulls in the MODULES file directly?